accepted/2020/net5/net5.md

```diff
@@ -0,0 +1,502 @@
+  requirements on both .NET Core SDK and VS insertions. If the logic were in a
+  separate package (we've talked about pushing it down to the framework at
+  various times) it'd be great to just expose that. With the logic in NuGet,
+  it's reasonable either way, just different tradeoffs. I'm amenable to exposing
+  a property function, but maybe we should go down the road of not doing it at
+  first.
+
+We're going to map the new entries as follows:
+
+| Framework      | Identifier    | Version | Profile |
+|----------------|---------------|---------|---------|
+| net48          | .NETFramework | 4.8     |         |
+| net5.0         | .NETCoreApp   | 5.0     |         |
+| net5.0-android | .NETCoreApp   | 5.0     | android |
+| net5.0-ios     | .NETCoreApp   | 5.0     | ios     |
+| net5.0-win     | .NETCoreApp   | 5.0     | win     |
+
+***Open Issue**. Please note that `net5.0`+ will map the TFI to `.NETCoreApp`.
+We need to announce this change so that package authors with custom .props and
+.targets are prepared. Link to DavKean’s doc on how to do it.*
+
+***Open Issue**. We should try to keep the TFI out of the .nuspec file. It seems
+NuGet uses the long form `.NETFramework,Version=4.5` in the dependency groups.
+We may want to change NuGet to allow the short form there as well and update our
+packaging tools to re-write to short name on pack.
+
+Specifically:
+
+* **We'll continue to use .NETCoreApp as the TFI**. This reduces the number of
+  pieces and build logic that needs to change in .NET SDK.
+
+* **net4x and earlier will continue to use .NETFramework as the TFI**. This
+  means that net4 and net5 aren't considered compatible by default, but the
+  compatibility will continue to be handled by the AssetTargetFallback in the
+  SDK/NuGet restore.
+
+* **We'll use the existing profile concept to encode the OS selection.** It's
+  worth noting that profiles today make things smaller whereas this use makes
+  things larger. We believe this can be handled in the NuGet layer by using this
+  rule for net5 and up. While that's not ideal, it reduces the number of
+  concepts we need to add.
+
+### TFMs are a closed set
+
+One question is whether third parties can extend the TFM space (i.e. the part
+after the dash) without having to rely on changes in NuGet/MSBuild. Due to the
+expansion of short name into TFI & TFM this would be non-trivial.
+
+We may open this up in the future, but for now the consensus was that we'd
+prefer to have the specific list of names that are expanded by NuGet.
+
+### What about .NET 10?
+
+Due to the fact that we're planning to bump the major version every year, we
+have to think about what will happen with version parsing in case of two digit
+version numbers, such as `net10`. Since `net10` already has a meaning (.NET
+Framework 1.0) we need to keep it that way. To avoid suprises, we'll by default
+use dottet version numbers in project templates to push developers towards being
```

Comment:
```suggestion
use dotnet version numbers in project templates to push developers towards being
```

Reply:
I don't think that this is a typo. Here the docs is referring to the "dot" (the `.`) in the version number, like demonstrated in the table below (e.g. `5.0` or `10.0`).